package-lock.json

```diff
@@ -1,8 +1,19392 @@
 {
   "name": "veilarbvedtaksstottefs",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Tror kanskje jeg ville droppet disse endringene  i package-lock siden det ikke trengs for denne brukerhistorien